vm/src/stdlib/bisect.rs

```diff
@@ -0,0 +1,182 @@
+pub(crate) use _bisect::make_module;
+
+#[pymodule]
+mod _bisect {
+    use std::convert::TryFrom;
+
+    use crate::function::OptionalArg;
+    use crate::slots::PyComparisonOp::Lt;
+    use crate::vm::VirtualMachine;
+    use crate::{ItemProtocol, PyObjectRef, PyResult};
+    use num_traits::ToPrimitive;
+
+    #[derive(FromArgs)]
+    struct BisectArgs {
+        #[pyarg(any, name = "a")]
```

Comment:
`#[pyarg(any)] foo: Bar` implies `name = "foo"`, you don't have to specify `name` for these.